### PR TITLE
Dashboard: Template Details UI follow up

### LINKED ISSUE #6637: Dashboard: Template Details UI follow up
## Feature Description

There are a few new things in the template details view for the dashboard from[ figma ](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=467%3A13745)that weren't doable in scope of #6599 because we lacked details from Design. 
This ticket is to complete that work after discussing with Design.  

### Buttons 
What are these labels? Are these only on hover? What should they actually say? 
|Screen Shot 2021-03-05 at 10 48 25 AM|Screen Shot 2021-03-05 at 10 48 21 AM|

## Grids
The grid seems skinnier than our current aspect ratio, is this intentional? We've implemented this to be responsive to the viewport size. What's the max width you want the small grid items and what is the max width you want the current page (large)? Is the goal to just have 3 rows always be the same total height of the active view page?

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria

Make updates necessary from the above questions.

### PR DESCRIPTION
**Pull request by maxyinger**
## Context
Just some polish updates to the template details page. Primarily around how the active page and grid are laid out.

## Summary
Updates sizing of grid relative to active page in template details page to always make sure the active page is 3 thumbnail pages high.

## Relevant Technical Choices
Basically went with an approach to statically position everything out regardless of stories getting rendered, then to measure that and render a story page within that predefined space. This approach lets the styling/dom do most of the layout work then just read what it has to say vs getting into crazy mathematical/logical operations.

## To-do
- Add labels on focus

## User-facing changes
Sizing of thumbnails and active page within template details page should be updated:
Screen Shot 2021-07-21 at 2 41 21 PM
Screen Shot 2021-07-21 at 2 41 09 PM

https://user-images.githubusercontent.com/35983235/126551614-f004ed9f-40a1-42c0-ae1d-587da9a165cc.mp4

## Testing Instructions
Go to dashboard -> templates -> click on any template -> navigate through templates & resize browser. Make sure everything looks solid

### QA
- go to dashboard
- explore templates
- hover over any template
- click `view details` button
- see that grid showcasing active page and thumbnails is updated
- click next/prev through templates as well as resize your browser and see that the story thumbnail/active page grid looks good

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6637

### CODE REVIEW

File: assets/src/dashboard/components/cardGallery/components.js [hunk 1]
```diff
@@ -17,95 +17,114 @@
...
+      ? css`
+          width: 26%;
+          margin: 0 7.1% 7.1% 0;
+        `
+      : css`
+          width: 19.74%;
+          margin: 0 5.26% 5.26% 0;
+        `}
+
+  :nth-last-child(-n + ${({ theme }) => theme.numRows}) {
```
Comment (BrittanyIRL) [inline assets/src/dashboard/components/cardGallery/components.js:127]: 💅  fancy!

**Review by BrittanyIRL — COMMENTED** [review]
this is just worlds better, thank you for doing so much refactor here, it makes more sense and the keyboard navigation is greatly improved. 

The detail page tests need some updates to pass, otherwise LGTM.

File: assets/src/dashboard/app/views/templateDetails/content/detailsGallery/index.js [hunk 2]
```diff
@@ -48,14 +54,23 @@ const DescriptionText = styled(Text)`
...
-  align-items: center;
-  ${({ rightPadding }) =>
-    rightPadding ? 'padding-right: 52px;' : 'padding-left: 52px;'}
+  position: absolute;
+  top: ${470 / 2}px;
+  ${({ alignLeft }) =>
+    alignLeft
+      ? css`
+          left: 0;
+          transform: translate(-187.5%, -50%);
```
Comment (samwhale) [inline assets/src/dashboard/app/views/templateDetails/content/detailsGallery/index.js:63]: From talk with max: 100% is to move arrow over an axis, then the extra 87.5% is to account for the margin.

Comment (samwhale) [reply assets/src/dashboard/app/views/templateDetails/content/detailsGallery/index.js:63]: The 50% is the vertical centering

**Review by samwhale — APPROVED** [review]
:yay-fox:

**Review by BrittanyIRL — APPROVED** [review]

**Pull request merged**